cypress/e2e/po/pages/fleet/gitrepo-create.po.ts

```diff
@@ -69,4 +74,13 @@ export class GitRepoCreatePo extends PagePo {
     gitRepoPaths() {
       return new ArrayListPo('[data-testid="gitRepo-paths"]');
     }
+
+    authSelect(selector: string) {
```

Comment:
So is it `select` without `create`? 🤔 
We should keep naming consistent with the model.